src/bundle/Resources/public/js/scripts/admin.location.bookmark.js

```diff
@@ -1,6 +1,28 @@
 (function (global, doc) {
     const SELECTOR_FORM = 'form[name="location_update_bookmark"]';
-    const updateBookmark = () => doc.querySelector(SELECTOR_FORM).submit();
+    const SELECTOR_BOOKMARK_CHECKBOX = '#location_update_bookmark_bookmarked';
+    const SELECTOR_BOOKMARK_LOCATION_INPUT = '#location_update_bookmark_location';
 
-    doc.querySelector('#location_update_bookmark_bookmarked').addEventListener('change', updateBookmark, false);
+    const submitBookmarkForm = () => doc.querySelector(SELECTOR_FORM).submit();
+    const updateBookmarkCheckbox = (bookmarked) => {
+        const checkbox = doc.querySelector(SELECTOR_BOOKMARK_CHECKBOX);
+
+        checkbox.checked = bookmarked;
+    }
+    const isCurrentLocation = (locationId) => {
+        const updateBookmarkLocationInput = doc.querySelector(SELECTOR_BOOKMARK_LOCATION_INPUT);
+        const currentLocationId = updateBookmarkLocationInput.value;
+
+        return parseInt(locationId, 10) === parseInt(currentLocationId, 10);
```

Comment:
This: `parseInt(currentLocationId, 10)` should be cached outside the function, unless it has a dynamic value.

Reply:
Why traverse DOM and read the attribute all the time? I think when needed is better.